R/categories.R

```diff
@@ -289,9 +291,11 @@ changeCategoryID <- function (variable, from, to) {
 
     ## Add new category
     newcat <- categories(variable)[[pos.from]]
+    if (newcat$id == newcat$numeric_value %||% FALSE) {
+        newcat$numeric_value <- to
```

Comment:
Agree that a code comment is good, but I don't think a user warning is necessary. If users have set their ids and numeric_values to be the same, they'll expect them to remain the same.